internal/transport/transport_test.go

```diff
@@ -780,10 +780,12 @@ func (s) TestGracefulClose(t *testing.T) {
 		go func() {
 			defer wg.Done()
 			str, err := ct.NewStream(ctx, &CallHdr{})
-			if err == ErrConnClosing {
-				return
-			} else if err != nil {
-				t.Errorf("_.NewStream(_, _) = _, %v, want _, %v", err, ErrConnClosing)
+			if err != nil {
+				select {
+				case <-ct.ctxDone:
```

Comment:
This needs a comment to explain what it's needed for.

Reply:
Done